perses/charts/values.yaml

```diff
@@ -226,3 +226,8 @@ perses:
     tls: []
   #    - secretName: ingress-cert
   #      hosts: []
+
+# -- By setting this to true, You will get some default dashboards
+greenhouse:
+  defaultDashboards:
+    enabled: true
```

Comment:
if we move comment above enabled i feel it will look better in README

Reply:
Thank you. fixed in 3159fa46b67c62f756a36c7247719f1d969c7fea